core/src/main/java/com/netflix/conductor/core/config/Configuration.java

```diff
@@ -105,6 +105,14 @@
     String EVENT_EXECUTION_PERSISTENCE_TTL_SECS_PROPERTY_NAME = "workflow.event.execution.persistence.ttl.seconds";
     int EVENT_EXECUTION_PERSISTENCE_TTL_SECS_DEFAULT_VALUE = 0;
 
+    String WORKFLOW_ARCHIVAL_TTL_SECS_PROPERTY_NAME = "workflow.archival.ttl.seconds";
+    int WORKFLOW_ARCHIVAL_TTL_SECS_DEFAULT_VALUE = 0;
+
+    String WORKFLOW_ARCHIVAL_DELAY_SECS_PROPERTY_NAME = "workflow.archival.delay.seconds";
```

Comment:
Exposing a configuration without an ability to actually configure sounds very counter-intuitive. I believe that it would be better to just use the async update delay property directly.

Reply:
Property `workflow.archival.delay.seconds` can be configured and it takes `getAsyncUpdateDelay()` as default value. In an ideal scenario, where the index delay queue size is near zero, this property can be same value as `getAsyncUpdateDelay()`. When the index delay queue increases due to the unavailability of worker threads/ES throttles then indexes might not be created for updating and so in those cases, this property can provide some buffer time for delays.